CHANGELOG.md

```diff
@@ -8,7 +8,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ## [Unreleased]
 
 ### Added
-
+- Luminex Xponent parser able to process output as pure csv file.
```

Comment:
This line is duplicated from below (assuming a merge conflict) - remove and we're all set to merge!